announcements/1.16.0.md

```diff
@@ -30,7 +30,7 @@ With the introduction of support for Android 13, Anbox Cloud now separates the `
 
 #### New bug report utility
 
-The Anbox [images](https://anbox-cloud.io/docs/ref/provided-images) include a new bug report tool called `anbox-bug-report`, which is extensible and collects information in an archive rather than a file. This method allows developers and support to easier inspect the information when debugging issues. See the [documentation](https://anbox-cloud.io/docs/howto/troubleshoot/collect-info) for more details.
+The Anbox [images](https://anbox-cloud.io/docs/ref/provided-images) include a new bug report tool called `anbox-bug-report`, which is extensible and collects information in an archive rather than a file. This method allows developers and support to easier inspect the information when debugging issues. See the [documentation](https://anbox-cloud.io/docs/howto/troubleshoot/landing) for more details.
```

Comment:
Given that these are older release notes, I'm not sure if we care, but:
Shouldn't these link to discourse instead?

```suggestion
The Anbox [images](https://discourse.ubuntu.com/t/provided-images/24185) include a new bug report tool called `anbox-bug-report`, which is extensible and collects information in an archive rather than a file. This method allows developers and support to easier inspect the information when debugging issues. See the [documentation](https://discourse.ubuntu.com/t/how-to-troubleshoot-anbox-cloud/17837) for more details.
```

Reply:
Yes, I used the docs link because the older link had that format and I was thinking there might be a reason behind it. But after checking around, it looks more like an inconsistency that crept in. So I have corrected this link. Although there are others directly linking to the website rather than discourse, I have not corrected those for now. I will do that as a separate cleanup exercise.